universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java

```diff
@@ -29,10 +31,16 @@ public Content htmlContent(DomContent... domContents) {
   }
 
   /**
-   * Returns a link tag for common CSS that should be included in the head of most other layouts.
+   * Returns a script tag that loads Tailwindcss styles and configurations common to all pages in
+   * the UAT.
+   *
+   * <p>This should be added to the end of the body of all layouts. Adding it to the end of the body
+   * allows the page to begin rendering before the script is loaded.
+   *
+   * <p>Adding this to a page allows Tailwindcss utility classes to be be usable on that page.
    */
-  protected Tag getCommonCssTag() {
-    return viewUtils.makeLocalCssTag("common");
+  protected Tag tailwindStyles() {
+    return viewUtils.makeLocalJsTag(WEBPACK_OUTPUT_FILENAME);
```

Comment:
Question (non-blocking): seems slightly odd to load the styles via a javascript tag. Does the JS just attach a very large `<style>` tag to the DOM?

Reply:
I completely agree!! I tried to find out a way to avoid this, but I don't know how to compile the Tailwindcss syntax without importing it into a `.js` file... 😕 Very open to suggestions.

Reply:
To answer your question of whether it attaches a very large `<style>` tag: Probably? But it's hard to tell exactly what's going on. This is what the generated `bundle.js` looks like: https://gist.github.com/natsid/b33435e43008cca08e6c2868d7bc8f07.

Reply:
Looked at this together

Reply:
For the record, our offline chat resulted in #235.